target/linux/ramips/dts/RAK633.dts

```diff
@@ -0,0 +1,117 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "ralink,RAK633", "RAK633";
```

Comment:
the board is not made by "ralink," please use the vendor name here

Reply:
Please use lower case for the compatible string. Use the `<vendor>,<boardname>` in userspace instead of `rak633`.